tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/RabbitMQ/RabbitMQConstants.cs

```diff
@@ -12,7 +12,9 @@ internal static class RabbitMQConstants
         internal const string IntegrationName = nameof(Configuration.IntegrationId.RabbitMQ);
         internal const string IBasicPropertiesTypeName = "RabbitMQ.Client.IBasicProperties";
         internal const string IDictionaryArgumentsTypeName = "System.Collections.Generic.IDictionary`2[System.String,System.Object]";
-        internal const string OperationName = "amqp.command";
+        internal const string AmqpCommand = "amqp.command";
+        internal const string ConsumerOperation = "amqp.process";
+        internal const string PublisherOperation = "amqp.send";
```

Comment:
Are these lines used at all in the product code? If they're only used in tests, maybe we can replace the test code with string literals?